R/run_eggla_gwas.R

```diff
@@ -508,6 +518,42 @@ run_eggla_gwas <- function(
         "--out", results_file
       ), collapse = " "))
 
+      if (!quiet) message(sprintf("[%s] Computing PLINK2 frequencies ...", basename(vcf)))
+      system(paste(c(
+        bin_path[["plink2"]],
+        "--vcf", vcf_file, "dosage=DS",
+        "--mach-r2-filter",
+        "--threads", threads,
+        "--freq",
+        if (file.exists(sprintf("%s.samples", basename_file))) c("--keep", sprintf("%s.samples", basename_file)),
+        "--silent",
+        "--out", results_file
+      ), collapse = " "))
```

Comment:
Allele frequencies report by PLINK2 using the same input as for GLM.

Reply:
I know it says outdates now but are we using the frequencies from this?

Reply:
Yes.

It's "outdated" because they were some changes in the file.

Reply:
I cannot find where it is used can you indicate please?

Reply:
See <https://github.com/mcanouil/eggla/pull/97#discussion_r1115480771>.